fabfile.py

```diff
@@ -1628,6 +1672,23 @@ def as_rc(ini_filename):
     return r
 
 
+@task
+def backup_piwik_data():
```

Comment:
Bad task name, as it does not really do what it says. This task seems to just fill a Jinja template and upload it to a server, it does not backup anything, right? Example of task name that could be used instead: upload_backup_script